sdk/appconfiguration/perf-tests/app-configuration/test/listSettings.spec.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { generateUuid } from "@azure/core-http";
+import { PerfStressOptionDictionary, executeParallel } from "@azure/test-utils-perfstress";
+import { AppConfigTest } from "./appConfigBase.spec";
+
+interface ListTestOptions {
+  count: number;
```

Comment:
Number of settings? Number of iterations? (some description/comment here would be usefu, or rename)

Reply:
added